src/js/breadcrumbs/breadcrumbs-model.js

```diff
@@ -0,0 +1,18 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone'], function (Backbone) {
+        var BreadcrumbsModel = Backbone.Model.extend({
+            defaults: {
+                'breadcrumbs': null,
+                'nav_aria_label': ''
```

Comment:
Nit: I'd call this something more semantic, e.g. ``label`` or ``title``. It isn't necessarily only going to be used by ARIA.